sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentSignatureType.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentSignatureType. */
 @Immutable
 public final class DocumentSignatureType extends ExpandableStringEnum<DocumentSignatureType> {
+
+    /**
+     * Creates a DocumentSignatureType object.
+     */
+    public DocumentSignatureType() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.